File: 14-longest-common-prefix/longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
       Arrays.sort(strs);
       String a="";
       for (int i=0;i<Math.min(strs[0].length(),strs[strs.length-1].length());i++){
        if (strs[0].charAt(i)==strs[strs.length-1].charAt(i)){
            a=a+strs[0].charAt(i);
        }else{
            break;
        }
       }
       return a;
    }
}

File: 14-longest-common-prefix/longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        strs.sort()
        s=""
        for i in range(0,min(len(strs[0]),len(strs[len(strs)-1]))):
            if(strs[0][i]==strs[len(strs)-1][i]):
                s=s+strs[0][i]
            else: break
        return s
